GOVERNANCE.md

```diff
@@ -1,98 +1,171 @@
-- Project: a collaboration effort, e.g. a subsystem, that is organized through the project creation process and approved by the CC.
-- Community Project: projects within the Node.js Foundation or in the ecosystem that contribute to the health of the Node.js project.
-- Adviser: a Collaborator within a Community Project elected to represent the Community Project on the CC.
-- Member: a person who participates in the development of the Community Committee through code or other artifacts.
-
-The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in all Community Committee documents are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
-
-For the current list of CC members, see the project
-[README.md](./README.md#community-committee-collaborators).
+## Collaborator Nominations
+
+Active Contributors of a CC initiative or working group can be
+nominated by another Collaborator to become a Collaborator of
+that initiative or working group. Contributors making
+significant, ongoing, and valuable contributions are also
+identified by the CC and their addition as Collaborators is
+discussed during the weekly CC meeting. On approval by the
+consensus process, the Contributor becomes a Collaborator and is
+given commit-access to the repo(s) relevant to their initiative.
+
+Modifications to the contents of the git repositor(y/ies) under
+the responsibility of the Collaborators are made on a
+collaborative basis as defined in the development process.
+Collaborators may opt to elevate significant or controversial
+modifications, or modifications that have not found consensus,
+to the CC for discussion by assigning the cc-agenda tag to a
+pull request or issue. The CC should serve as the final arbiter
+where required. The CC will maintain and publish a list of
+current Collaborators by Project, as well as a development
+process guide for Collaborators and Contributors looking to
+participate in the effort.
+
+Collaborators may request Emeritus status when they find themselves inactive. The
+CC also periodically reviews the Collaborator list to identify inactive Collaborators.
+Past Collaborators are typically given Emeritus status. Emeriti may request that
+the CC restore them to active status.
+
+### Onboarding
+
+All collaborators added as CommComm members should be on-boarded in a timely manner
+and be given write access to the repository.
+
+## Consensus Seeking Process
+
+The TSC follows a [Consensus Seeking][] decision making model as described by
+the [TSC Charter][].
+
+[collaborators-discussions]: https://github.com/orgs/nodejs/teams/collaborators/discussions
+[Consensus Seeking]: https://en.wikipedia.org/wiki/Consensus-seeking_decision-making
+[TSC Charter]: https://github.com/nodejs/TSC/blob/master/TSC-Charter.md
```

Comment:
Same comment: shouldn't this point to the newly created CommComm charter?

Reply:
+1

Reply:
again sorry for copy paste error

Reply:
@MylesBorins Honestly, apologizing for making the tiniest mistake while helping on our work... You should be chastized indeed :smiley: